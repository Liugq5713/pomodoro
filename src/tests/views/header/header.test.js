import React from 'react'
import Header from 'views/header'
import renderer from 'react-test-renderer'
test('should have header every page', () => {
  const component = renderer.create(<Header />)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
