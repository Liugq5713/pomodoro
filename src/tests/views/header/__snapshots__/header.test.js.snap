// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should have header every page 1`] = `
<nav
  aria-label="main navigation"
  className="navbar has-background-black-ter"
  role="navigation"
>
  <div
    className="navbar-brand"
  >
    <div
      className="navbar-item is-paddingless"
      href="https://github.com/Liugq5713"
    >
      <img
        alt=""
        className="svg"
        height="28"
        src="logo.svg"
        width="112"
      />
    </div>
    <a
      aria-expanded="false"
      aria-label="menu"
      className="navbar-burger"
      href="https://github.com/Liugq5713"
      role="button"
    >
      <span
        aria-hidden="true"
      />
      <span
        aria-hidden="true"
      />
      <span
        aria-hidden="true"
      />
    </a>
  </div>
</nav>
`;
